Vostok.ServiceDiscovery.Telemetry.Hercules/HerculesServiceDiscoveryEventsFactory.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Linq;
+using JetBrains.Annotations;
+using Vostok.Hercules.Client.Abstractions.Events;
+using Vostok.ServiceDiscovery.Telemetry.Event;
+
+namespace Vostok.ServiceDiscovery.Telemetry.Hercules
+{
+    /// <summary>
+    /// Converts <see cref="HerculesEvent"/>s written by <see cref="HerculesServiceDiscoveryEventsSender"/> back to <see cref="ServiceDiscoveryEvent"/>s.
+    /// </summary>
+    [PublicAPI]
+    public static class HerculesServiceDiscoveryEventsFactory
+    {
+        [NotNull]
+        public static ServiceDiscoveryEvent From([NotNull] HerculesEvent herculesEvent)
```

Comment:
глагол точно должен быть в названиях методов)

```suggestion
        public static ServiceDiscoveryEvent CreateFrom([NotNull] HerculesEvent herculesEvent)
```